randomkick.py

```diff
@@ -135,6 +135,18 @@ async def kick_user(delay, *, warn):
                 await asyncio.sleep(random.randint(7, 10))
             await bot.send_message(GROUP, 'Oh darn! That was close 😅')
 
+        warn_message = await bot.get_message(GROUP, event.id)
+        # message deleted or expired
+        if not warn_message:
```

Comment:
Not completely sure if `warn` ever ends up being false, but should probably account for that

```suggestion
        warn_message = await bot.get_message(GROUP, event.id) if warn else None
        # message deleted or expired
        if warn and not warn_message:
```